.github/workflows/end2end.yaml

```diff
@@ -381,7 +385,9 @@ jobs:
         with:
           push: true
           context: ./tests/ctst
-          build-args: CTST_TAG=${{ env.CTST_TAG }}
+          build-args: |
+            CTST_TAG=${{ env.CTST_TAG }}
+            DRCTL_TAG=v1.0.2
```

Comment:
nit: we may want to add workflow variable to hold the DRCTL version (`v1.0.2`) which is used in multiple places